README.md

```diff
@@ -1,180 +1,75 @@
-
-Swift-DocC outputs a machine-readable archive of the compiled documentation.
-This archive contains _render JSON_ files, which fully describe the contents
-of a documentation page and can be processed by a renderer such as
-[Swift-DocC-Render](https://github.com/apple/swift-docc-render).
-
-For more in-depth technical information about Swift-DocC, please refer to the
-project's technical documentation:
-
-- [`SwiftDocC` framework documentation](https://apple.github.io/swift-docc/documentation/swiftdocc/)
-- [`SwiftDocCUtilities` framework documentation](https://apple.github.io/swift-docc/documentation/swiftdoccutilities/)
-
-### Related Projects
-
-  - As of Swift 5.5, the [Swift Compiler](https://github.com/apple/swift) is able to 
-    emit _Symbol Graph_ files as part of the compilation process.
-    
-  - [SymbolKit](https://github.com/apple/swift-docc-symbolkit) is a Swift package containing
-    the specification and reference model for the _Symbol Graph_ File Format.
-  
-  - [Swift Markdown](https://github.com/apple/swift-markdown) is a 
-    Swift package for parsing, building, editing, and analyzing 
-    Markdown documents. It includes support for the Block Directive elements
-    that Swift-DocC's tutorial files rely on.
-    
-  - [Swift-DocC-Render](https://github.com/apple/swift-docc-render) 
-    is a web application that understands and renders
-    Swift-DocC's _render JSON_ format.
-    
-  - [Xcode](https://developer.apple.com/xcode/) consists of a suite of
-    tools that developers use to build apps for Apple platforms.
-    Beginning with Xcode 13, Swift-DocC is integrated into Xcode
-    with support for building and viewing documentation for your framework and
-    its dependencies.
-  
-## Getting started with developing `docc`
+## Getting Started with DocC
 
 `docc` is the command line interface (CLI) for Swift-DocC and provides
 support for converting and previewing DocC documentation.
 
-### Prerequisites
-
-DocC is a Swift package. If you're new to Swift package manager,
-the [documentation here](https://swift.org/getting-started/#using-the-package-manager)
-provides an explanation of how to get started and the software you'll need
-installed.
-
-DocC requires Swift 5.5 which is included in Xcode 13.
-
-### Build
-
-1. Checkout this repository using:
+There are multiple ways you can make use of DocC depending on your use case:
 
-    ```bash
-    git clone https://github.com/apple/swift-docc.git
-    ```
+**1. For standalone documentation:**
 
-2. Navigate to the root of the repository with:
+If you have Xcode installed, it's recommended to generate documentation using the `xcrun` command.
+You can get DocC working by invoking `xcrun docc` in your terminal.
 
-    ```bash
-    cd swift-docc
-    ```
+Swift-DocC is also included in the toolchain for both macOS and Linux.
 
-3. Finally, build DocC by running:
+To see instructions on how to use DocC from the CLI run
+```
+docc --help
```

Comment:
I saw folks asking questions about whether we expect folks to use `docc` on it's own as a CLI tool, and I think @ethan-kusters said that not really. See https://swift-open-source.slack.com/archives/C04B8HURKB4/p1691608726725189. 

I think this line is valid and useful, with a gotcha that it's then user's responsibility to make sure that the symbol graph is built. Perhaps if the user does not have Xcode available, it's best to suggest that while they can run `docc` directly, if they're just _using_ docc to build docs, they probably want the Swift DocC Plugin and Swift package manager?